packages/jest-runner/src/run_test.js

```diff
@@ -55,10 +55,10 @@ async function runTestInternal(
 
   let testEnvironment = config.testEnvironment;
 
-  if (customEnvironment) {
+  if (customEnvironment && customEnvironment[0]) {
```

Comment:
This was failing Flow because it was passing an array instead of a string(and before the flow type was more relaxed)

From what I understand this only workder because at some point in `getTestEnviromnent` we are doing `jest-environment-${env}` which if env was an array(`jest-envrionment-${['my-env']}`), it would just get stringified as `jest-environment-my-env`. Same story when it does `require.resolve(`${prefix}${fileName}`)`.


Does this change make sense? From what I can think you can only pass one env per docblock

Reply:
>Does this change make sense? From what I can think you can only pass one env per docblock

Yeah, it does! We might want to allow multiple envs in the future (to test in both node and the browser), but for now this is correct 🙂

Reply:
I might just add a `@FlowFixMe` because it seems that even though the type says `Array<string> | string` it seems to be a string when passing the docblock. 

I'll investigate more later today because I have to head out.